sender.go

```diff
@@ -323,6 +323,30 @@ func (s *LineSender) Int64Column(name string, val int64) *LineSender {
 	return s
 }
 
+// Long256Column adds a 256-bit unsigned integer(long256) column
+// value to the ILP message.
+//
+// Column name cannot contain any of the following characters:
+// '\n', '\r', '?', '.', ',', ”', '"', '\\', '/', ':', ')', '(', '+',
+// '-', '*' '%%', '~', or a non-printable char.
+func (s *LineSender) Long256Column(name, val string) *LineSender {
```

Comment:
nit: would be nice to accept `math/big.Int` instead of a raw string.

Reply:
We tried the implementation with `math/big.Int` but there are a few unnecessary steps we need to perform on both the user's progam(like `main.go`)and in the go-questDB-client. 
Example:
`main.go` file
```go
    // rest of the code.
	hexaString := "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
	bigVal := new(big.Int)
	bigVal.SetString(hexaString, 16)
	err = sender.
		Table("trades").
		Symbol("name", "test_ilp1").
		Long256Column("value", bigVal).
		At(ctx, time.Now().UnixNano())
    // rest of the code.
```
& 
`sender.go` file
```go
func (s *LineSender) Long256Column(name string, val *big.Int) *LineSender {
    // rest of the code
    
    // adding the prefix and suffix.
	s.lastErr = s.writeStrValue("0x"+val.String()+"i", false)
    
    // rest of the code
}
```
Another limitation is that the `big.NewInt()` takes int64 which is also limited to 64 bits.
To solve this we also have `fmt.Sscan()` which takes number(int, long, float, etc)string and converts it into an any desired format.
Example:
```go
	i := new(big.Int)
	_, err := fmt.Sscan("115792089237316195423570985008687907853269984665640564039457584007913129639935", i)
    	if err != nil {
		log.Println("error scanning value:", err)
	} else {
		fmt.Println(i)
	}
```
Output: `115792089237316195423570985008687907853269984665640564039457584007913129639935`
Again, for sending `i` or output to the questDB we would need to perform addition steps of converting it into a hexadecimal and add prefix and postfix.

This is the reason, we implemented string. It offers a simple and straightforward solution.

Reply:
There is [`SetBytes()`](https://pkg.go.dev/math/big#Int.SetBytes) method which can be used to initialize the integer without dealing with a string.